### DISCUSSION TITLE
Allow syncing all repos of Organization/User

### DISCUSSION THREAD

**Discussion by Timeraa** [BetaHuhn/repo-file-sync-action#155] [Ideas]
I would like to see the possibility to sync files between all repos of my Organization / User.
This could be used if you set/add `repos: PreMiD/*` as an example. It should allow a RegExp to filter out unwanted ones for example

**Comment by BetaHuhn** [comment]
Related to #127. 

I agree, supporting RegExp in repo names as well as branch names (#83) would be a great feature to have.

Upon checking on the status of #127, I noticed @jpoehnelt has already implemented parts of this in his [fork](https://github.com/jpoehnelt/repo-file-sync-action). @jpoehnelt, what's the status of this?

If anyone is willing to create a PR (maybe use @jpoehnelt's fork as inspiration), happy to review and get this merged.

**Reply by Timeraa** [reply]
I might try to look into it and create a PR with it.

**Comment by CalvinRodo** [comment]
I'm also interested in this.

It would be nice if this feature also allowed for an ignore list to be used.

**Comment by ReenigneArcher** [comment]
There is a similar action that does this: https://github.com/marketplace/actions/copy-files-to-other-repositories

But overall this action if far better... unfortunately I need the ability to sync to all repos as it's a lot of work to maintain this one as new repos are added.

The action I linked allows you to select the repository based on topics. So you can add a topic like `sync-python` or something, and it will sync to repos that have that topic. Then it allows you to exclude specific repos if you want.

Hope you can add this!

**Comment by thomascrowley** [comment]
I have just created #238 but I realise now this is the same thing. Has there been any progress with this?
